ansible/roles/monasca/templates/monasca-log-transformer/monasca-log-transformer.json.j2

```diff
@@ -0,0 +1,18 @@
+{
+    "command": "/usr/share/logstash/bin/logstash --log-in-json --log /var/log/kolla/monasca/monasca-log-transformer.log -f /etc/logstash/conf.d/log-transformer.conf",
+    "config_files": [
+        {
+            "source": "{{ container_config_directory }}/log-transformer.conf",
+            "dest": "/etc/logstash/conf.d/log-transformer.conf",
```

Comment:
It might be nice to eventually be able to drop files into a conf.d directory in the kolla config, like we do for fluent.

Reply:
Good idea, added to TODO list